### PR TITLE
Add support

### PR DESCRIPTION
**Pull request by maysakanoni**
**What?**
-Add support page
- logs a call
- logs a help request

**Notes**
-Add more tests
-Log case notes when the endpoint has been added
image

### CODE REVIEW

File: src/pages/add-support/[residentId]/index.js [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+					</div>
+					<div class="govuk-grid-column-three-quarters-from-desktop">
+					{errorsExist && <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+							<h2 class="govuk-error-summary__title" id="error-summary-title">
+								There is a problem
+							</h2>
+							<div class="govuk-error-summary__body">
+								<ul class="govuk-list govuk-error-summary__list">
+									<li>
+										<a href="#passport-issued-error">You have not completed the form</a>
```
Comment (kosiakkatrina) [inline src/pages/add-support/[residentId]/index.js:178]: This probably shouldn't link to `#passport-issued-error`

File: src/components/SupportTable/SupportTable.jsx [hunk 2]
```diff
@@ -35,8 +36,7 @@ export default function SupportTable({helpRequests}) {
 							<td className="govuk-table__cell">{helpRequest.callbackRequired && <>Callback</>}</td>
 							<td className="govuk-table__cell">{helpRequest.helpRequestCalls?.length}</td>
 							<td className="govuk-table__cell govuk-table__cell--numeric">
-								{/* <a href="/helpcase-profile/1/call-starter">View</a>  */}
-								<a href="#">View</a> 
+								<a href="/helpcase-profile/1/call-starter">View</a>
```
Comment (kosiakkatrina) [inline src/components/SupportTable/SupportTable.jsx:39]: Should we leave 'view' pointing to nothing until we someone starts working on that page?

**Review by kosiakkatrina — APPROVED** [review]
If we're leaving the follow up requested question and if it's meant to not update if 'No' is selected then it's okay. Seems to work, nice!
I'd probably like to separate the add-support file into smaller components, but that could probably be future refactoring

File: src/components/SupportTable/SupportTable.jsx (hunk 2 above)
Comment (maysakanoni) [reply src/components/SupportTable/SupportTable.jsx:39]: I guess so

**Pull request merged**